chart/templates/deployment.yaml

```diff
@@ -22,6 +22,14 @@ spec:
         image: {{ .Values.scannerImage}}:{{ .Values.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
         env:
+        - name: GOMEMLIMIT
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.memory
+        - name: GOMAXPROCS
```

Comment:
This has to be an integer, right? I don't know if `limits.cpu` has to be an int. Also, I'm curious why this was added